### PR TITLE
initial version

### PR DESCRIPTION
**Pull request by orviwan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
-# feature-content-size
\ No newline at end of file
+# feature-content-size
+
+An example application which demonstrates how to utilize the ContentSize API
+in Pebble applications.
+
+<kbd>![Screenshot](/screenshots/anim.gif)</kbd>
+
+[ContentSize Documentation](https://developer.pebble.com/docs/c/)
```
Comment (cjwilliams) [inline README.md:8]: Maybe link directly to the ContentSize docs once you know the URL?

File: src/content_size.c [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      break;
+    case PreferredContentSizeLarge:
+      text_layer_set_font(s_text_layer, fonts_get_system_font(FONT_KEY_GOTHIC_24_BOLD));
+      text_layer_set_text(s_text_layer, "Content Size\nLarge");
+      break;
+    case PreferredContentSizeExtraLarge:
+      text_layer_set_font(s_text_layer, fonts_get_system_font(FONT_KEY_GOTHIC_28_BOLD));
+      text_layer_set_text(s_text_layer, "Content Size\nExtra Large");
+      break;
+    case NumPreferredContentSizes:
```
Comment (cjwilliams) [inline src/content_size.c]: default?

File: wscript [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    ctx.load('pebble_sdk')
+
+def build(ctx):
+    ctx.load('pebble_sdk')
+
+    build_worker = os.path.exists('worker_src')
+    binaries = []
+
+    for p in ctx.env.TARGET_PLATFORMS:
+        ctx.set_env(ctx.all_envs[p])
```
Comment (cjwilliams) [inline wscript]: This is an old wscript, do not use it

**Review by cjwilliams — CHANGES_REQUESTED** [review]

**Comment by orviwan** [comment]
Ready for another review @cjwilliams

**Review by cjwilliams — APPROVED** [review]

**Comment by martijnthe** [comment]
Does it make sense to also add a Rocky.js example for `rocky.userPreferences.contentSize` in this repo?
In terms of "finding your way" in the examples, I think it makes sense to have both Rocky+C in the same repo. It's also a good way to lure C devs into trying Rocky if they haven't already :D

**Pull request merged**